src/core/src/bootstrap/EnvLayer.py

```diff
@@ -430,26 +430,26 @@ def write_with_retry_using_temp_file(file_path, data, mode='w'):
                         raise Exception("Unable to write to {0} (retries exhausted). Error: {1}.".format(str(file_path), repr(error)))
 
         @staticmethod
-        def delete_files_from_dir(dir_name, file_identifier_list, raise_if_delete_failed=False):
-            """ Clears all files from given dir. NOTE: Uses file_identifier_list to determine the content to delete """
-            for file_identifier in file_identifier_list:
-                files_to_delete = glob.glob(str(dir_name) + "/" + str(file_identifier))
+        def delete_from_dir(dir_name, identifier_list, raise_if_delete_failed=False):
+            """ Clears all files/dirs from given dir. NOTE: Uses identifier_list to determine the content to delete """
+            for identifier in identifier_list:
+                items_to_delete = glob.glob(os.path.join(str(dir_name), str(identifier)))
```

Comment:
@rane-rajasi - notes: not using os.path.join made this code OS-environment sensitive.